tor/README.md

```diff
@@ -0,0 +1,19 @@
+Using cnd over Tor
+==================
+
+`cnd` may be used to do atomic swaps anonymously across the Tor network. In order to do so you will need to do the following:
+
+0. Install Tor
+
+Either use your package manager or see [here](https://2019.www.torproject.org/docs/tor-doc-unix).
+
+1. Configure an onion service using the Tor run file.
+
+You can use the exaple `./torrc` and run `tor` using `sudo /usr/bin/tor --defaults-torrc tor-service-defaults-torrc -f torrc --RunAsDaemon 0`
+
+2. Set the onion address in the cnd config file.
+
+Once you have run `tor` for the first time get the onion address from the `hostname` file (if you used the `tor` invocation above this will be in `/var/lib/tor/hidden_service/hostname`).
+See `./cnd.toml` for an example config file.
+
+3. Before starting cnd to do a swap you will need a running Tor instance. Currently we use the default SOCKS5 Tor port 9050.
```

Comment:
What does the dependency on SOCKS5 this mean for running `cnd` on MacOS and Windows?

Reply:
> What does the dependency on SOCKS5 this mean for running `cnd` on MacOS and Windows?

I have no clue how SOCKS5 works on MacOS, its UNIX so I assumed it would work the same as Linux. The default port is set by Tor. I did not add configuration for the port to cnd since we have not run into the problem of needing it yet.

We don't support Windows anyways so ...

Or am I missing something in your question?

Reply:
> Or am I missing something in your question?

Nope! :)
Just wanted to make sure that what we are doing here doesn't lock us out from supporting swaps on several operating systems.

I assume if our precondition is "configure a hidden service with Tor on port 9939" then this is also gonna work on Windows, however it is done over there (if it is even possible).